stories/5-components/Navigation/AuDropdown.stories.js

```diff
@@ -57,20 +58,22 @@ const Template = (args) => ({
       @hideText={{this.hideText}}
       @alert={{this.alert}}
       @onClose={{this.onClose}}
-      role="menu"
     >
       <AuButton @skin="link" @icon="switch" role="menuitem">
         Switch profile
       </AuButton>
       <AuButton @skin="link" @icon="logout" role="menuitem">
         Afmelden
       </AuButton>
-    </AuDropdown>`,
+    </AuDropdown>
+    {{! template-lint-enable require-context-role }}
+  `,
   context: args,
 });
 
 const TemplateSeparator = (args) => ({
   template: hbs`
+    {{! template-lint-disable require-context-role }}
```

Comment:
It might be a good idea to yield button and link components that are pre-wired to do the right thing. Apps will also have to disable this rule as well which isn't ideal.

I'll create an issue so I don't forget to look at it.

Reply:
Sure, that would be a more robust solution but a bit out of scope of my current assignment.